examples/stm32h7-example/Core/Src/cli_setup.c

```diff
@@ -0,0 +1,118 @@
+/*
+ * cli_setup.c
+ *
+ *  Created on: Jul 7, 2023
+ *      Author: NeusAap
+ */
+
+
+// STD LIBS
+#include <stdio.h>
+#include <stdlib.h>
+
+// STM files
+#include "usart.h"
+#include "stm32h7xx_it.h"
+
+// Own CLI headers
+#include "cli_setup.h"
+#include "cli_binding.h"
+
+// UART buffers
+uint8_t UART_CLI_rxBuffer[UART_RX_BUFF_SIZE] = {0};
```

Comment:
This can also be static, right? Please check all variables in this file and `cli_binding.c`, looks like they all can be static.